libs/langchain/tests/integration_tests/.env.example

```diff
@@ -33,5 +16,7 @@ POWERBI_TABLE_NAME=_test_table_name_here
 POWERBI_NUMROWS=_num_rows_in_your_test_table
 
 
-# MongoDB Atlas Vector Search
-MONGODB_ATLAS_URI=your_mongodb_atlas_connection_string
\ No newline at end of file
+# astra db
+ASTRA_DB_API_ENDPOINT=https://your_astra_db_id-your_region.apps.astra.datastax.com
```

Comment:
should this be in community .env.example?

Reply:
it's in both.
We also need it here because the LLM cache impls are in the langchain package, not langchain-community